Gems/Atom/RPI/Code/Include/Atom/RPI.Reflect/Asset/AssetUtils.h

```diff
@@ -133,21 +135,10 @@ namespace AZ
             template<typename AssetDataT>
             Data::Asset<AssetDataT> LoadCriticalAsset(const AZStd::string& assetFilePath, TraceLevel reporting)
             {
-                bool apConnected = false;
-                AzFramework::AssetSystemRequestBus::BroadcastResult(
-                    apConnected, &AzFramework::AssetSystemRequestBus::Events::ConnectedWithAssetProcessor);
-                if (apConnected)
-                {
-                    AzFramework::AssetSystem::AssetStatus status = AzFramework::AssetSystem::AssetStatus_Unknown;
-                    AzFramework::AssetSystemRequestBus::BroadcastResult(
-                        status, &AzFramework::AssetSystemRequestBus::Events::CompileAssetSync, assetFilePath);
-                    if (status != AzFramework::AssetSystem::AssetStatus_Compiled && status != AzFramework::AssetSystem::AssetStatus_Unknown)
-                    {
-                        AssetUtilsInternal::ReportIssue(reporting, AZStd::string::format("Could not compile asset '%s'", assetFilePath.c_str()).c_str());
-                        return {};
-                    }
-                }
+                TryToCompileAsset(assetFilePath, reporting);
```

Comment:
It's a redundant check because CompileAssetSync also checks for an AP connection and returns back "Unknown" if the AP isn't connected, so I removed it.